Dockerfile-express

```diff
@@ -0,0 +1,9 @@
+FROM node
+COPY mock_data/ /VisualizationG5/mock_data/
+COPY express/ /VisualizationG5/express/
+WORKDIR /VisualizationG5/express/
+RUN rm -rf node_modules/
```

Comment:
You might want to consider using the [dockerignore file](https://docs.docker.com/engine/reference/builder/#dockerignore-file) for the node_modules folder to avoid copying it.